.github/workflows/integration-opensuse-leap.yml

```diff
@@ -1,14 +1,5 @@
 name: integration-opensuse-leap
-
-on:
-  push:
-    branches:
-      - openSUSE-3004
-  pull_request:
-    branches:
-      - openSUSE-3004
-  workflow_dispatch:
```

Comment:
IMO it is nice to have a way to manually trigger the tests, but I have to admit I haven't used it in a long time. Does it hurt keeping this?

Reply:
Agree. I does not hurt keeping it :+1: